File: CS/B217847Example/Form1.cs
using System;
using System.Linq;
using System.Windows.Forms;
using System.Collections.Generic;

namespace B217847Example
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            DataTableHelper.GetDataTable(dtTest);
        }

        private void OnFilterAdapterRowFilterChanged(object sender, EventArgs e)
        {
            gridView1.ActiveFilterCriteria = filterAdapter1.RowCriteria;
        }

        private void OnApplyFilterButtonClick(object sender, EventArgs e)
        {
            filterControl1.ApplyFilter();
        }
    }
}

File: VB/B217847Example/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Linq
Imports System.Windows.Forms
Imports System.Collections.Generic

Namespace B217847Example
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
			DataTableHelper.GetDataTable(dtTest)
		End Sub

		Private Sub OnFilterAdapterRowFilterChanged(ByVal sender As Object, ByVal e As EventArgs) Handles filterAdapter1.RowFilterChanged
			gridView1.ActiveFilterCriteria = filterAdapter1.RowCriteria
		End Sub

		Private Sub OnApplyFilterButtonClick(ByVal sender As Object, ByVal e As EventArgs) Handles simpleButton1.Click
			filterControl1.ApplyFilter()
		End Sub
	End Class
End Namespace
